packages/ui5-proxy-middleware/test/base/utils.test.ts

```diff
@@ -465,8 +468,10 @@ describe('utils', () => {
         });
 
         test('HTML is modified and response is sent', async () => {
+            const projectRoot = process.cwd();
```

Comment:
The test is not very helpful since you mock the file access. Please add a folder with a sample project structure under test/fixtures and then use that for testing instead of mocking fs.

Reply:
Done with c0bdb3cbd8e4eeb655a2256e312f16db7ccefa37